flow/scripts/cts.tcl

```diff
@@ -98,8 +98,17 @@ if { [info exists ::env(SKIP_GATE_CLONING)] } {
   append additional_args " -skip_gate_cloning"
 }
 
+
+if { [info exists ::env(EQUIVALENCE_CHECK)] } {
+    write_eqy_verilog 4_before_rsz.v
```

Comment:
Should this be written in the `objects` folder?

Reply:
lets merge this PR for now and deal with this later. I do not have a strong opinion on where this files goes.